main.go

```diff
@@ -100,3 +116,65 @@ func renderPage(page string) string {
 	}
 	return result
 }
+
+type pageCache struct {
+	diskv *diskv.Diskv
+}
+
+// CachedPage is a page stored in the diskv cache.
+type CachedPage struct {
+	// The page content.
+	Page string
```

Comment:
perhaps rename this to Content as well (for consistency with the recent variable rename?